src/core/coreav.cpp

```diff
@@ -113,7 +113,8 @@ bool CoreAV::anyActiveCalls()
 
 bool CoreAV::isCallVideoEnabled(uint32_t friendNum)
 {
-    assert(calls.contains(friendNum));
+    bool contains = calls.contains(friendNum);
+    assert(contains);
     return calls[friendNum].videoEnabled;
```

Comment:
The solution is to make ```contains``` const then, not create variables. Semantically, contains MUST be const or the code is insane.